src/k8s-engine/k8s.ts

```diff
@@ -82,6 +82,13 @@ export interface KubernetesBackend extends events.EventEmitter {
   /** Progress for the current action. */
   progress: Readonly<KubernetesProgress>;
 
+  /**
+   * Whether debug mode is enabled. If the implementation has any ability to
+   * add --debug flags, or enable any kind of debug mode, this property should
+   * enable or disable that.
```

Comment:
```suggestion
   * Whether debug mode is enabled. If this is set, the implementation should
   * emit extra debug logging if possible.
```
(Make it clearer that debug mode should only affect logging, and introduce any other change in behaviour like spawning in a debugger.)